### DISCUSSION TITLE
Is it possible to disable image embedding?

### DISCUSSION THREAD

**Discussion by diecknet** [giscus/giscus#955] [Q&A]
Hey everyone,

is it possible to disable image embedding with Giscus? I want to prevent inappropriate images.
Or is it possible to use some kind of moderation?

**Comment by laymonage** [comment]
I think it's currently possible if you use a [custom theme](https://github.com/giscus/giscus/blob/main/ADVANCED-USAGE.md#data-theme) and have something like

```css
img {
  display: none;
}
```

**Reply by diecknet** [reply]
I like that idea. I'll try that soon
